Add new "no element handle" rule (#40)
playwright-community/eslint-plugin-playwright: README.md

await page.click('button');
```

### `no-element-handle`

> Disallow the use of `page.$` or `page.$$` element handle.

Reviewer: ```suggestion
Disallow the creation of element handles with `page.$` or `page.$$`.
```

> 👎 Examples of <span style="color:red">**incorrect**</span> code for this rule:

Reviewer: ```suggestion
Examples of **incorrect** code for this rule:
```

```js
// Element Handle
const buttonHandle = await page.$('button');
await buttonHandle.click();

// Element Handles
const linkHandles = await page.$$('a');
```

> 👍 Example of <span style="color:green">**correct**</span> code for this rule:

Reviewer: ```suggestion
Example of **correct** code for this rule:
```
